### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by Vishnu-Aithal**
Add - Cart

Preview - https://deploy-preview-2--vishnu-store.netlify.app/pages/cart.html

### CODE REVIEW

File: pages/cart.html [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+            <div class="card__dismiss">
+              <i class="fas fa-times clr-white"></i>
+            </div>
+            <div class="card__body p-2">
+              <div class="card__img-wrapper">
+                <img
+                  class="card__img"
+                  src="https://picsum.photos/200/300"
+                  alt="Card Header Image"
+                  srcset
```
Comment (Maianki) [inline pages/cart.html:161]: You have not given any value to srcset attribute. Either remove it or give it a value.

**Review by Maianki — APPROVED** [review]
Great job Vishnu. Just one suggestion everything else looks good.

File: pages/cart.html (hunk 1 above)
Comment (Vishnu-Aithal) [reply pages/cart.html:161]: ok thanks for reviewing

File: stylesheets/cart.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  max-height: 100%;
+  border-right: solid 2px var(--color-secondary-light);
+  overflow-y: auto;
+}
+
+.cart-info {
+  width: 300px;
+  grid-column: 4/7;
+  grid-row: 1/7;
+
```
Comment (Developers-codz) [inline stylesheets/cart.css:37]: remove extra line

**Review by Developers-codz — APPROVED** [review]
Everything else look good to merge

File: stylesheets/cart.css (hunk 2 above)
Comment (Vishnu-Aithal) [reply stylesheets/cart.css:37]: ok Thanks for the Review

**Pull request merged**